src/components/dashboard/sidebar/SidebarCard.jsx

```diff
@@ -1,35 +1,34 @@
-/* eslint-disable react/require-default-props */
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Card } from '@edx/paragon';
 
-import ButtonWithLink from '../../layout/ButtonWithLink';
-
-import './styles/SidebarCard.scss';
+import './styles/_SidebarCard.scss';
```

Comment:
Since this is now imported from `src/index.scss`, I don't think it needs to be imported here.